apps/selvbetjening-ui/src/inntektsjustering/2-inntekt-til-neste-aar/InntektsjusteringInntektTilNesteAar.tsx

```diff
@@ -28,6 +29,8 @@ export const InntektsjusteringInntektTilNesteAar = () => {
         switch (alder) {
             case Alder.ATTEN_TIL_FEMTI_SEKS:
                 return <AttenTilFemtiSeksAarSkjema />
+            case Alder.FEMTI_SYV_TIL_SEKSTI:
+                return <FemtiSyvTilSekstiAarSkjema />
         }
     }
```

Comment:
Det er det her som jeg må se på om mer hensiktsmessig, for nå blir det en del dobbelt opp. Får kanskje se an hvordan designet blir for de 2 andre skjemaene